apps/meteor/server/models/raw/LivechatContacts.ts

```diff
@@ -195,4 +199,37 @@ export class LivechatContactsRaw extends BaseRaw<ILivechatContact> implements IL
 			options,
 		).toArray();
 	}
+
+	async isContactActiveOnPeriod(visitorId: string, period: string): Promise<boolean> {
+		const query = {
+			'channels.visitorId': visitorId,
+			'activity': period,
+		};
+
+		const foundActiveContact = (await this.countDocuments(query)) > 0;
```

Comment:
Think we can just use JS help and return the number here, so people can use `if isContactActiveOnPeriod` and let js convert it to bool, avoiding the second promise. wdyt?